clusterloader2/testing/hpa/README.md

```diff
@@ -0,0 +1,18 @@
+## HPA controller load test
+
+Assumptions:
+- Cluster master noded are accessible via SSH.
+- The `/metrics` endpoint in the kube-controller-manager (KCM) can be accessed by anonymous users.
+  `--authorization-always-allow-paths` can be useful to achieve this.
+
+HPA load test perform the following steps:
+- Create replicasets, services, HPAs in a set of namespaces.
+  - The namespaces used here are managed by CL2.
+  - The number of HPAs in each namespace can be specified using `CL2_HPA_PER_NAMESPACE`.
+- Periodically measure the HPA controller workqueue depth in each KCM instance
+  - In each scrape, we use the max value among them since the leader will have more items in the workqueue.
+  - We keep tracking the smallest value over time.
```

Comment:
Can you clarify this - what's the significance of tracking the smallest value over time?